zephyr/Kconfig

```diff
@@ -56,4 +56,14 @@ config ZEPHYR_DP_SCHEDULER
 	  DP modules can be located in dieffrent cores than LL pipeline modules, may have
 	  different tick (i.e. 300ms for speech reccognition, etc.)
 
+config CHECK_INIT_PRIORITIES
```

Comment:
This is not the right way. We can just set this to 'n' in the in sof/app/prj.conf 

But we can't do this until the we update to latest Zephyr mainline and take in this patch
```
commit 058f256ea140026e46e458dd51036ea2c1f81b2a
Author: Fabio Baltieri <fabiobaltieri@google.com>
Date:   Fri Mar 10 17:26:35 2023 +0000

    Kconfig.zephyr: check priorities by default 
```

Then you'll get errors like this:
https://github.com/thesofproject/sof/actions/runs/5973406935/job/16205619385?pr=8095

Reply:
I thought about that and we should do it as soon as possible. Is there any arguments against update?

Reply:
@pjdobrowolski no obstacles, we can do one immediately. I just merged a 2200+ commit update yesterday, so we are very close to Zephyr mainline already so I don't expect many (new) issues beside this priority level check.

We need to investigate the priority check warnings as well and why those fail for us.

Reply:
> I thought about that and we should do it as soon as possible.

We must do it in the _same commit_ where we update the Zephyr manifest and get this new feature. Neither before nor after to make sure every SOF commit builds.

If you need a local workaround in the mean time then `west config build.cmake-args -- -DCONFIG_CHECK_INIT_PRIORITIES=n` is probably the most convenient?

> I just merged a 2200+ commit update yesterday,

This sort of number can be deceiving; these could be almost all in drivers that we'll never use.